tests/test_check_obsfiles_ert_yaml.py

```diff
@@ -0,0 +1,105 @@
+from datetime import datetime
+
+import os.path
+
+# import pytest
+
+import yaml
+
+
+def read_ert_obs(ert_obs_file_name):
+    # Reading ERT file
+    assert os.path.exists(ert_obs_file_name) == 1
+    a_ert_file = open(ert_obs_file_name, "r")
```

Comment:
Always use `with` to open a file.

Like so:

```python
with open(xyz,"r") a fh:
    for line in .......
```
When you leave the with block you automatically close the file.

Don't forget documentation.